.changelog/1872.txt

```diff
@@ -0,0 +1,7 @@
+```release-note:bug
+serverinstall/ecs: validate memory and cpu values
+```
+
+```release-note:bug
+plugin/aws/ecs: validate memory and cpu values
+```
```

Comment:
I think these should be in one block, mostly because I think go-changelog only supports one block per file?

Reply:
You can have multiple entries in a single go-changelog txt file! https://github.com/hashicorp/waypoint/blob/main/.github/CHANGELOG_GUIDE.md#multiple-entries

Reply:
![bitmoji](https://sdk.bitmoji.com/render/panel/849e65c4-2dc2-4877-8981-2dc04080868a-8cb64c44-6a64-4f62-9642-460dd37039d6-v1.png?transparent=1&palette=1&width=246)